pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -262,8 +263,7 @@ func (s *DiscoveryServer) StreamAggregatedResources(stream ads.AggregatedDiscove
 					adsLog.Debugf("ADS:LDS: ACK %s %s %s %s", peerAddr, con.ConID, discReq.VersionInfo, discReq.ResponseNonce)
 					continue
 				}
-				// too verbose - sent immediately after EDS response is received
-				adsLog.Debugf("ADS:LDS: REQ %s %v", con.ConID, peerAddr)
+				adsLog.Infof("ADS:LDS: REQ %s %v", con.ConID, peerAddr)
```

Comment:
Also info is too noisy !

Reply:
As the test result shows, LDS REQ is only sent on first connection

Reply:
I do not think it is just on first connection - it will be sent on every reconnection of Envoy -> Pilot which happens every 30 mins as per the default config? So all the time one or the other Envoy keeps reconnecting.

Reply:
reconnects happen at the period of 30m